### PR TITLE
Release 1.1.0

### PR DESCRIPTION
**Pull request by turtlemoji**
- Revamp UI (multiple fixes, major refactor to the layout and styles)
- Fix error handling in transaction component

### CODE REVIEW

**Review by FoxTheSin — APPROVED** [review]

File: src/client/components/app/Footer/index.tsx
Comment (karelianpie) [inline src/client/components/app/Footer/index.tsx:137]: We should avoid using the index for keys https://reactjs.org/docs/lists-and-keys.html#keys

I think the link is a stable id for this
```suggestion
          <StyledIconLink href={social.link} target="_blank" key={social.link}>
```

File: src/client/components/app/Navbar/index.tsx
Comment (karelianpie) [inline src/client/components/app/Navbar/index.tsx:124]: Is `&nbsp;/&nbsp;` needed? Perhaps better to create a component with margin around /

File: src/client/components/app/Navbar/index.tsx
Comment (karelianpie) [inline src/client/components/app/Navbar/index.tsx:120]: nit: I think we don't need to check for length
```suggestion
  const secondTitleEnabled = !!subTitle;
```

File: src/client/components/app/Navbar/index.tsx
Comment (karelianpie) [inline src/client/components/app/Navbar/index.tsx:133]: question: Will this just appear next to one another? Is that the intention?

File: src/client/components/app/Navigation/NavSidebar.tsx
Comment (karelianpie) [inline src/client/components/app/Navigation/NavSidebar.tsx:91]: Is this needed?

File: src/client/components/app/Navigation/NavTabbar.tsx
Comment (karelianpie) [inline src/client/components/app/Navigation/NavTabbar.tsx:18]: 👍

File: src/client/components/app/Settings/ThemeBox.tsx
Comment (karelianpie) [inline src/client/components/app/Settings/ThemeBox.tsx:36]: 👍

File: src/client/components/app/SummaryCard/index.tsx
Comment (karelianpie) [inline src/client/components/app/SummaryCard/index.tsx:13]: Use `CardContent` directly?

File: src/client/components/app/Transactions/components/TxTokenInput.tsx
Comment (karelianpie) [inline src/client/components/app/Transactions/components/TxTokenInput.tsx:167]: Remove?

File: src/client/components/app/VaultDetail/VaultDetailPanels.tsx
Comment (karelianpie) [inline src/client/components/app/VaultDetail/VaultDetailPanels.tsx:86]: Use `TabPanel` directly

File: src/client/components/app/VaultDetail/VaultDetailPanels.tsx
Comment (karelianpie) [inline src/client/components/app/VaultDetail/VaultDetailPanels.tsx:88]: Use `Tabs` directly

File: src/client/components/common/Card/CardElement/index.tsx
Comment (karelianpie) [inline src/client/components/common/Card/CardElement/index.tsx:37]: Is this using the new yearn-web-lib? If so, we should have a Typography for h3

File: src/client/containers/Layout/index.tsx
Comment (karelianpie) [inline src/client/containers/Layout/index.tsx:198]: These new lines are probably not needed

File: src/client/routes/IronBank/index.tsx
Comment (karelianpie) [inline src/client/routes/IronBank/index.tsx:190]: Remove new line

File: src/client/routes/IronBank/index.tsx
Comment (karelianpie) [inline src/client/routes/IronBank/index.tsx:272]: Not needed

File: src/client/routes/Labs/index.tsx
Comment (karelianpie) [inline src/client/routes/Labs/index.tsx:350]: Remove

File: src/client/routes/Labs/index.tsx
Comment (karelianpie) [inline src/client/routes/Labs/index.tsx:416]: Remove

File: src/client/themes/classic/index.ts
Comment (karelianpie) [inline src/client/themes/classic/index.ts:83]: Is needed?

File: src/client/themes/classic/index.ts
Comment (karelianpie) [inline src/client/themes/classic/index.ts:99]: Remove?

**Review by karelianpie — APPROVED** [review]
Huge PR, went through all the files and only spotted minor stuff.

Trusting that the styles are looking good, only reviewed the code itself.

File: src/client/components/app/Footer/index.tsx
Comment (turtlemoji) [reply src/client/components/app/Footer/index.tsx:137]: I don't think it is a stable id, since it is a normal string that can be repeated pretty easily without any checks. maybe social.link-index would do if you don't like to let only the index

File: src/client/components/app/Navbar/index.tsx
Comment (turtlemoji) [reply src/client/components/app/Navbar/index.tsx:124]: I would leave nbsp since it is the real letter spacing related to that type of font and the current font-size. the margins will break the correct spacing if you change the font or size.

File: src/client/components/app/Navbar/index.tsx
Comment (turtlemoji) [reply src/client/components/app/Navbar/index.tsx:120]: true, I would just leave it for another pr for now

File: src/client/components/app/Navbar/index.tsx
Comment (turtlemoji) [reply src/client/components/app/Navbar/index.tsx:133]: yes.
image

File: src/client/components/app/Navigation/NavSidebar.tsx
Comment (turtlemoji) [reply src/client/components/app/Navigation/NavSidebar.tsx:91]: Nope, just a vestige from testing. We should add this things on a ticket to fix later

File: src/client/components/app/Navigation/NavTabbar.tsx
Comment (turtlemoji) [reply src/client/components/app/Navigation/NavTabbar.tsx:18]: Yep, this is needed

File: src/client/components/app/Settings/ThemeBox.tsx
Comment (turtlemoji) [reply src/client/components/app/Settings/ThemeBox.tsx:36]: Just vertical whitespace for readability

File: src/client/components/app/SummaryCard/index.tsx
Comment (turtlemoji) [reply src/client/components/app/SummaryCard/index.tsx:13]: Yes we can change this, we have some of these around the app since we were not against these wrappers in some cases or just vestiges of old refactors. I would create another PR to fix them all tho

File: src/client/components/app/Transactions/components/TxTokenInput.tsx
Comment (turtlemoji) [reply src/client/components/app/Transactions/components/TxTokenInput.tsx:167]: I would leave it for another pr too and just focus on technical stuff on this one

File: src/client/components/app/VaultDetail/VaultDetailPanels.tsx
Comment (turtlemoji) [reply src/client/components/app/VaultDetail/VaultDetailPanels.tsx:88]: Idem

File: src/client/components/common/Card/CardElement/index.tsx
Comment (turtlemoji) [reply src/client/components/common/Card/CardElement/index.tsx:37]: Nope, not being used. Also I would not put an h3 on card element, since an element can have another elements inside. I discussed with pentacle about it today since:
telegram-cloud-photo-size-1-5093984801542744590-y

File: src/client/containers/Layout/index.tsx
Comment (turtlemoji) [reply src/client/containers/Layout/index.tsx:198]: Not needed perse, just more vertical whitespacing. I like it to diferentiate things and readability:
image
OR
image

File: src/client/routes/IronBank/index.tsx
Comment (turtlemoji) [reply src/client/routes/IronBank/index.tsx:190]: Lets create another pr too

File: src/client/themes/classic/index.ts
Comment (turtlemoji) [reply src/client/themes/classic/index.ts:83]: technically not

File: src/client/themes/classic/index.ts
Comment (turtlemoji) [reply src/client/themes/classic/index.ts:99]: I think we will need to reable it to define contrast type of the theme. But could be removed in the meantime, in the other pr I mentioned tho

File: src/client/components/app/Navbar/index.tsx
Comment (turtlemoji) [reply src/client/components/app/Navbar/index.tsx:120]: > Huge PR, went through all the files and only spotted minor stuff.
> 
> Trusting that the styles are looking good, only reviewed the code itself.

reviewed at:


> feat: revamp UI [WEB-1381] (

reviewed at #536

File: src/client/components/app/Navigation/NavTabbar.tsx
Comment (turtlemoji) [reply src/client/components/app/Navigation/NavTabbar.tsx:18]: > Huge PR, went through all the files and only spotted minor stuff.
> 
> Trusting that the styles are looking good, only reviewed the code itself.

yep, tested and reviewed at #536

**Comment by turtlemoji** [comment]
> Huge PR, went through all the files and only spotted minor stuff.
> 
> Trusting that the styles are looking good, only reviewed the code itself.

yep, tested and reviewed at #536

**Comment by turtlemoji** [comment]
@karelianpie let's wait for @xgambitox gambito approval and we can discuss and make the new PR with your comments :).
I would like just to keep vertical whitespaces for readability. I agree with the other ones.

File: src/client/components/app/Footer/index.tsx
Comment (karelianpie) [reply src/client/components/app/Footer/index.tsx:137]: All good, let's leave as is

File: src/client/containers/Layout/index.tsx
Comment (karelianpie) [reply src/client/containers/Layout/index.tsx:198]: I still think it's readable enough without the new lines, but up to you. If you want to leave those, I think it's not the end of the world :)

**Pull request merged**
